webcomponents/vectortile-view-component/src/adminunits.html

```diff
@@ -34,38 +34,38 @@ <H1>administrative unit NL </H1>
 
     <script>
```

Comment:
Blijven deze test bestanden (`adminunits.html` en `tile-samples.html`) buiten het uiteindelijke artifact? Niet dat ze per ongeluk in de bundle worden meegepacked.

Reply:
Examples verhuist naar example directory, dat is een betere locatie. Kwamen niet mee in de npm build en worden ook niet meegekopieerd in de docker build. (Ook index.html gaat niet mee in de dockerbuild.)